src/common/attachments/uploadAttachmentsDialog.component.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  useTheme,
+} from '@mui/material';
+import AwsS3, { AwsBody } from '@uppy/aws-s3';
+import Uppy, { Meta, UppyFile } from '@uppy/core';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+import ProgressBar from '@uppy/progress-bar'; // Import the ProgressBar plugin
+import { Dashboard } from '@uppy/react';
+import React from 'react';
+import { usePostAttachmentMetadata } from '../../api/attachments';
+
+const MAX_FILE_SIZE_MB = 100;
+const MAX_FILE_SIZE_B = MAX_FILE_SIZE_MB * 1048576;
```

Comment:
This is actually doing more than 100 MB, this would be the conversion between mebibytes to bytes, instead of megabytes.

Also a note to make an issue if this will be configurable. The backend uses an env variable `ATTACHMENT__MAX_SIZE_BYTES` defaulted to 100000000 (bytes was used as it was the format the limits actually used).